drivers/spi/spi_ll_stm32.c

```diff
@@ -807,6 +807,38 @@ static bool spi_buf_set_in_nocache(const struct spi_buf_set *bufs)
 }
 #endif /* CONFIG_SOC_SERIES_STM32H7X */
 
+#ifdef CONFIG_SPI_STM32F7_ERRATA_BUSY
+static void wait_spi_dma_not_busy(const struct device *dev)
+{
+	const struct spi_stm32_config *cfg = dev->config;
+	SPI_TypeDef *spi = cfg->spi;
+	int64_t start = k_uptime_ticks();
+
+	while (true) {
```

Comment:
Can we use  [WAIT_FOR](https://github.com/zephyrproject-rtos/zephyr/blob/3b5fd5fc929bdc958c1acd435726d28c0ecf3f8e/include/zephyr/sys/util.h#L703) ?

Reply:
Cool, didn't know this exists.
done